runner/src/mill/runner/CliImports.scala

```diff
@@ -0,0 +1,8 @@
+package mill.runner
+
+import scala.util.DynamicVariable
+
+/**
+ * Hold additional runtime dependencies given via the `--import` cli option.
+ */
+private[runner] object CliImports extends DynamicVariable[Seq[String]](Seq.empty)
```

Comment:
Huh I never knew `extends DynamicVariable` was a thing, but I guess it works

Reply:
Yeah, I first had it as a `val` in the companion object of `MillBuildBootstrap`, but `acyclic` wasn't happy with that. So I moved it and tried to avoid all that nesting, without using a `val` in a new `package object` and ended up with this compact thing.